ivc/tests/add.rs

```diff
@@ -341,7 +346,7 @@ pub fn heavy_test_simple_add() {
         }
     }
 
-    type Config = StandardConfig<
+    type Config<const N_COL: usize, const N_FSEL: usize> = StandardConfig<
```

Comment:
Why this change?

Reply:
It is adding noise, but ok... I would simply remove this.

Reply:
It will be generalized and moved out of this file (into the library) 5 PRs later, to be used with different circuits.

Here it was originally introduced because I was experimenting with several setups at the same time.